bootstrap_test.py

```diff
@@ -280,6 +280,44 @@ def test_consistent_range_movement_true_with_rf1_should_fail(self):
     def test_consistent_range_movement_false_with_rf1_should_succeed(self):
         self._bootstrap_test_with_replica_down(False, rf=1)
 
+    """
+    Test creating a KS with RF > N on multi DC doesn't break bootstrap
+    @jira_ticket CASSANDRA-16296 CASSANDRA16411
+    """
+    def test_rf_gt_nodes_multidc_should_succeed(self):
```

Comment:
Nit: I think doctring should be the first statement in the function's body. There is also a missed hyphen in the second ticket id:
```suggestion
    def test_rf_gt_nodes_multidc_should_succeed(self):
        """
        Test creating a KS with RF > N on multi DC doesn't break bootstrap
        @jira_ticket CASSANDRA-16296 CASSANDRA-16411
        """
```